code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -58,7 +58,7 @@
 		to_chat(src, src.playstyle_string)
 		to_chat(src, "<B><span class ='notice'>You are not currently in the same plane of existence as the station. Ctrl+Click a blood pool to manifest.</span></B>")
 		src << 'sound/misc/demon_dies.ogg'
-		mind.current.verbs += /mob/living/simple_animal/slaughter/proc/slaughterWhisper
+		mind.current.verbs += /mob/living/simple_animal/slaughter/verb/slaughterWhisper
```

Comment:
well my goal here was to fix an issue where the whisper thing wasn't popping up for demons..do i need to just remove this or add something else in to make sure its added to the mob properly?